src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +39,16 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" to={"/" + (query ? `?q=${query}#${searchInputId}` : "")}>
```

Comment:
Why not use [`createPath`](https://github.com/ReactTraining/react-router/blob/master/docs/API.md#createpathpathorloc-query)?